reactioncommerce/reaction
# #3354 Fix fulfilling part of a multi-merchant order removes other parts of order

Resolves #3354 

Fixes order been deleted when a part of a multi-merchant order is `packed` in the Order dashboard.

## Steps to Test

1. Setup a multi-merchant marketplace (primary shop + at least one merchant shop)
2. Enable the merchant shop(s)
3. Setup shipping and Stripe payments in the primary shop
4. Setup stripe payments in the merchant shop
5. Create and publish products from each shop
6. As a guest, add products from each merchant to your cart and checkout
7. Fulfill a single merchant's part of the order, from either the merchant or the primary shop.
8. Observe that the other parts of the order are not deleted when items are marked as packed
